qldbdriver/transaction.go

```diff
@@ -39,7 +39,7 @@ type transaction struct {
 	commitHash   *qldbHash
 }
 
-func (txn *transaction) execute(ctx context.Context, statement string, parameters ...interface{}) (*Result, error) {
+func (txn *transaction) execute(ctx context.Context, statement string, parameters ...interface{}) (Result, error) {
```

Comment:
Nit: I don't think the internal transaction needs to return Result instead of *result.

Reply:
Addressed.